test/index.js

```diff
@@ -17,259 +16,256 @@ function getLimitdClient(address) {
 }
 
 describe('options validation', () => {
-  it ('should fail if event is not specified', () => {
-    plugin.register(null, {
-      type: 'user',
-      limitd: getLimitdClient(),
-      extractKey: EXTRACT_KEY_NOOP
-    }, err => {
+  it('should fail if event is not specified', async () => {
+    try {
+      await plugin.register(null, {
+        type: 'user',
+        limitd: getLimitdClient(),
+        extractKey: EXTRACT_KEY_NOOP
+      });
+      expect(false, 'this should not be reached').to.be.true;
```

Comment:
This will be caught by the try catch i think (not sure), you will endup in the catch bellow, anyway I think that expect.fail is clearer for this use case

Reply:
Agree, but at least will fail with another error.